src/classes/LVL_LevelAssign_BATCH.cls

```diff
@@ -34,7 +34,12 @@
 * @description Batch class processes all Levels against Target objects, and updates any changes.
 */
 public without sharing class LVL_LevelAssign_BATCH implements Database.Batchable<sObject>, Database.Stateful {
-    
+    /** @description Custom Settings record that defines Levels settings */
+    @TestVisible
+    private static Levels_Settings__c levelsSettings = UTIL_CustomSettingsFacade.getLevelsSettings();
+
+    @TestVisible
+    private static final String DATETIME_SOQL_FORMAT = 'yyyy-MM-dd\'T\'HH:mm:ss\'Z\'';
     /** @description The query for the batch process to run on.*/
     private String strSoql;
```

Comment:
I know this is mostly out of the scope of this particular WI, but this is a small class. I think we should fix some of the glaring style issues:
* Remove hungarian notation
* Proper case data types
* Upper case SOQL keywords